core-blocks/pullquote/theme.scss

```diff
@@ -4,9 +4,11 @@
 	color: $dark-gray-600;
 
 	cite,
-	footer {
+	footer,
+	.citation {
```

Comment:
These styles are enqueued on the front-end, but we'll never have `.citation` on the front-end display of this block. Conversely, there won't ever be a `cite` in the editor to match.

I think we may want this in `editor.scss` instead? I'm not entirely sure what our practice here is with `theme.scss` and editor-specific styles (cc @jasmussen ).